packages/cli/src/commands/run.js

```diff
@@ -0,0 +1,88 @@
+import fs from 'fs'
+import path from 'path'
+
+import babelRequireHook from '@babel/register'
+import Listr from 'listr'
+import VerboseRenderer from 'listr-verbose-renderer'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from 'src/lib'
+import c from 'src/lib/colors'
+
+babelRequireHook({
+  extends: path.join(getPaths().api.base, '.babelrc.js'),
+  extensions: ['.js', '.ts', '.tsx', '.jsx'],
+  only: [getPaths().api.base],
+  plugins: [
+    [
+      'babel-plugin-module-resolver',
+      {
+        alias: {
+          src: getPaths().api.src,
+        },
+      },
+    ],
+  ],
+  ignore: ['node_modules'],
+  cache: false,
+})
+
+const { db } = require(path.join(getPaths().api.lib, 'db'))
+
+const runScript = async (scriptPath, scriptArgs) => {
+  const script = await import(scriptPath)
+  await script.default({ db, args: scriptArgs })
+}
+
+export const command = 'run <name>'
+export const description = 'Run your script'
+export const builder = (yargs) => {
+  yargs
+  .positional('name', {
+    description: 'The name of the script to run',
+    type: 'string',
+  }).strict(false).epilogue(
+    `Also see the ${terminalLink(
+      'Redwood CLI Reference',
+      'https://redwoodjs.com/docs/cli-commands#up'
+    )}`
+  )
+}
+
+export const handler = async (args) => {
+  const {name, ...scriptArgs} = args
+  const scriptPath = path.join(getPaths().api.scripts, `${name}.js`)
```

Comment:
```suggestion
  const scriptPath = path.join(getPaths().api.scripts, `${name}`)
```
I think if we take the `.js` out, we get typescript support fore free!

Reply:
Is there a way I can check the file exists with this change? Because `if (!fs.existsSync(scriptPath)) {` checks for exact path rather than modules.

Reply:
Maybe use `require.resolve` wrapped in a try/catch?

Reply:
Yep that worked!